packages/picasso/src/TableCell/styles.ts

```diff
@@ -19,9 +18,8 @@ const regularCellStyles = {
 
 const compactCellStyles = {
   ...baseCellStyles,
-  padding: '0.25rem 0.5rem',
+  padding: '0.0625rem 0.5rem',
```

Comment:
```suggestion
  padding: `${rem(1px) 0.5rem`,
```

when using unusual numbers in rem, we are using `rem` util function, so it is clear to everybody on first view what number it actually holds

Reply:
got it 🙂  thank you